data/data/azure/vnet/outputs.tf

```diff
@@ -2,13 +2,13 @@ output "cluster-pip" {
   value = azurerm_public_ip.cluster_public_ip.ip_address
 }
 
-output "network_id" {
-  value = data.azurerm_virtual_network.cluster_vnet.id
-}
+#output "network_id" {
```

Comment:
why the comment rather than deleting this?

Reply:
It slipped in. Removed.